src/components/TargetDetails/parseUtils.tsx

```diff
@@ -118,14 +118,18 @@ export const parseContent = (text: string | undefined) => {
   });
 };
 
-export const parseExamples = (example: string) => {
-  const lines = splitByNewLine(example);
-  let content;
-  if (lines) {
-    content = lines.map(parseContent);
-  } else {
-    content = parseContent(example);
+export const parseExamples = (example: string | string[]) => {
+  let lines: string[] | undefined = [];
+  if (typeof example === 'string') {
```

Comment:
I don't really like using typeof if we can avoid it but I don't think this instance is avoidable.

Reply:
I agree, but that seems to be a side effect of wanting this function to deal with strings and string arrays simultaneously. Alternatively, we could use a `try/catch` statement — attempting to run a `foreach` on `example`, then treating it as a string in the `catch` statement. I kind of like that approach.

Reply:
That would make the function look something like this:

```js
export const parseExamples = (example: string | string[]) => {
  let lines: string[] | undefined = [];
  let content;
  try {
    example.forEach(e => {
      const splits = splitByNewline(e);
      if (splits && lines) {
        lines = lines.concat(splits);
      }
    });
  } catch (e) {
    lines = splitByNewLine(example);
  } finally {
     content = lines ? lines.map(parseContent) : parseContent(example as string);
  }

  return content 
}
```